compose/nginx_site.conf

```diff
@@ -1,8 +1,17 @@
 server {
+    listen 80;
     index index.php;
     server_name php-docker.local;
+    error_log  /var/log/nginx/error.log;
```

Comment:
is it good to write log files in docker images?

Reply:
nevermind, it was already there, just moved to the top ...

Reply:
if you want, you are able to make a volume mount to the log dir an than you have persistent access to the log file

Reply:
okay 👍  I guess/hope docker users are aware of that^^

Reply:
@tobybatch maybe we should document where logfiles are stored?

Reply:
i have removed the logging paths, because its running in foreground of the docker container and is printed also to the stdout where a user can fetch the logs